docs/src/conventions.md

```diff
@@ -0,0 +1 @@
+# Conventions
```

Comment:
Is this file intended for a future chapter? Or maybe a predecessor to `database/conventions.md` that accidentally got left in?

Reply:
Looks like an accidental file. Will remove